### PR TITLE
Add TSlint

### LINKED ISSUE #15: Add TS linter
Add a TS linter

### PR DESCRIPTION
**Pull request by nerder**
# Description

I add the TSlint to the project and create couple of task in order to use the linter from the CLI.

Fixes #15 

## Type of change

Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# How Has This Been Tested?

run `npm run lint` and check if there are errors or running `npm run lint:fix` to fix the autofixable 

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by ghost** [comment]
we should add precommit hooks directly!

File: src/controllers/FlatController.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ export class FlatController {
 
   @Path("hello")
   @GET
-  public async helloWorld(): Promise<String> {
-    return 'Hello World!';
+  public async helloWorld(): Promise<string> {
+    return "Hello World!";
```
Comment (ghost) [inline src/controllers/FlatController.ts:9]: do we want dobule quotes then?

**Review by ghost — CHANGES_REQUESTED** [review]

File: src/controllers/FlatController.ts (hunk 1 above)
Comment (oscard0m) [reply src/controllers/FlatController.ts:9]: This is a new discussion with ESLint ;), I would suggest to create an issue for that ;)

**Comment by oscard0m** [comment]
Let's create an issue for the hooks @juanchaur ! 💪

**Comment by juanchaur** [comment]
issue created: https://github.com/dominguezcelada/rent-flat-scraper/issues/32

File: src/controllers/FlatController.ts (hunk 1 above)
Comment (juanchaur) [reply src/controllers/FlatController.ts:9]: Allright, I'll approve it but I'll add soon the single quote rule 😈

**Review by juanchaur — APPROVED** [review]

**Pull request merged**